scripts/run_experiment.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+#SBATCH --partition=ephemeral
+#SBATCH --qos=ephemeral
+#SBATCH --ntasks=1
+#SBATCH --mem-per-cpu=8G
+#SBATCH --cpus-per-task=1
+#SBATCH --job-name=ADAM_EXPERIMENT
+#SBATCH --output=data/slurm_logs/%x-%j.out    # %x-%j means JOB_NAME-JOB_ID.
+#SBATCH --requeue
+#SBATCH --exclude="saga01,saga02,saga03,saga04,saga05,saga06,saga07,saga08,saga10,saga11,saga12,saga13,saga14,saga15,saga16,saga17,saga18,saga19,saga20,saga21,saga22,saga23,saga24,saga25,saga26,gaia01,gaia02"
+# The excluded nodes above have slower CPUs on the cluster. This list targets our execution at the CPUs with the highest
+# base & boost frequency
+
+set -euo pipefail
+
+source "$1"/etc/profile.d/conda.sh
+conda activate "$2"
```

Comment:
I'm not sure we strictly need these lines? The Slurm job inherits the conda setup from your bash session when you run this using sbatch. That includes the currently activated environment.

Reply:
You're correct -- I'm used to configuring the jobs for within Pegasus mostly where this is strictly necessary. I'll remove it from this script.